src/pages/home/report/ReportActionCompose/index.js

```diff
@@ -255,6 +261,7 @@ class ReportActionCompose extends React.Component {
     }
 
     componentWillUnmount() {
+        removeListenKeyDown(this.keydownListener);
```

Comment:
We should perform this only if `keydownListener` is not null
```suggestion
        if (this.keydownListener) {
            removeListenKeyDown(this.keydownListener);
```